public/app/features/plugins/admin/helpers.ts

```diff
@@ -198,3 +214,10 @@ export const sortPlugins = (plugins: CatalogPlugin[], sortBy: Sorters) => {
 
   return plugins;
 };
+
+function groupByPluginId(errors: PluginError[]): Record<string, PluginError | undefined> {
```

Comment:
**Nit:** maybe calling it something like `groupErrorsByPluginId` instead would reveal a bit more what it is doing